src/test/java/lesson8Person/PersonTest.java

```diff
@@ -0,0 +1,43 @@
+package lesson8Person;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class PersonTest {
+
+    private Person person1 = new Person("Patryk", "Jelowicki", 1994, Month.December, 5,
```

Comment:
Skoro w każdym z testów potrzebujesz innego obiekty klasy Person, to twórz te obiekty bezpośrednio w testach. W formie pól trzymaj rzeczy, które są wykorzystywane w wielu testach, a ich utworzenie jest "ciężkie".